src/cf-modals/src/cf-modals.less

```diff
@@ -0,0 +1,91 @@
+/* ==========================================================================
+   Capital Framework
+   Modal Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
+// `$color-` variables referenced in comments are from 18F's U.S. Web Design Standards
+// https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
+
+// .o-modal
+@modal-bg-outer:              @black;
+@modal-bg-inner:              @white;
+@modal-action-bar:            @gray-5;
+@modal-border-top:            @green;
+
+// Import external dependencies
+
+@import (less) '../../cf-core/src/cf-core.less';
+@import (less) '../../cf-icons/src/cf-icons.less';
+
+.o-modal {
+   z-index: 99999;
+   display: none;
+
+   &__visible {
+     display:block;
+   }
+}
+
+.o-modal_backdrop {
+   position: fixed;
+   width: 100%;
+   height: 100%;
+   top: 0;
+   left: 0;
+   background-color: @modal-bg-outer;
+   opacity: 0.5;
+   z-index: 99999;
+}
+
+.o-modal_offset {
+   position: absolute;
+   top: 5%;
+   width: 100%;
+   z-index: 999999;
+}
+
+.o-modal_content {
+   position: relative;
+   width: 60%;
+   margin: 0 auto;
+   min-width: 200px;
+   max-width: 270px;
+}
+
+@media only all and (min-width:37.5625em) {
+   .o-modal_content {
+       max-width: 630px;
+       max-height: 700px;
+       overflow: auto;
+   }
+}
+
+.o-modal_inner {
+   border-top: 3px solid @modal-border-top;
+   background-color: @modal-bg-inner;
+   padding: 30px;
+}
+
+.o-modal_actionbar {
+   background: @modal-action-bar;
+   padding: 10px 30px;
+}
+
+.o-modal_inner h1 {
```

Comment:
I don't see an `h1` in the example. Is this necessary? Related, if it is, we should add a classname to the heading to keep with our Atomic principles.